.gitignore

```diff
@@ -3,4 +3,6 @@
 *.o
 dist/
 .stack-work/
-*.DS_Store
\ No newline at end of file
+<<<<<<< HEAD
+*.DS_Store
```

Comment:
Sorry that's my mistake. *.DS_store occurs on macs. I didn't mean to commit it, it's just an annoying thing that always shows up in git!